### ISSUE TITLE
fit_shape in PSFPhotometry

### ISSUE THREAD

**Issue by aromalsankaran** [astropy/photutils#1967]
We are doing PSF photometry on point sources in JWST images and notice that the PSF flux_fit is roughly independent of the fit_shape parameter (whether it is (3,3) or (9,9)). We would like to know if the fit_shape only decides the region used for initial PSF fitting and later it includes the entire region when calculating flux? We would like to get the PSF flux for a smaller region and later use aperture correction to get the total flux if possible.

**Comment by larrybradley** [comment]
> We would like to know if the fit_shape only decides the region used for initial PSF fitting and later it includes the entire region when calculating flux?

Yes, exactly.  The `flux` parameter is a scale factor that is applied to the input model (which is assumed to be normalized).

> We would like to get the PSF flux for a smaller region and later use aperture correction to get the total flux if possible.

You could do this by creating a PSF model with the fitted parameters, evaluate the model over the small region in which you are interested, and then sum the resulting values.

**Comment by larrybradley** [comment]
Please reopen if you have further questions.
